react/AppIcon/test/AppIcon.spec.js

```diff
@@ -0,0 +1,108 @@
+'use strict'
+/* eslint-env jest */
+
+import React from 'react'
+import { configure, shallow } from 'enzyme'
+import Adapter from 'enzyme-adapter-react-15'
+
+import AppIcon from '../'
+
+configure({ adapter: new Adapter() })
```

Comment:
Jest is configured for all the tests to use the React 15 adapter. 

https://github.com/cozy/cozy-ui/blob/98ed325f89ff04193df8cf6614a0e1250b28b55c/test/jestsetup.js#L2-L4

There is no need to `configure()` in each test ;)